x-pack/plugins/security/server/authorization/privileges/feature_privilege_builder/cases.ts

```diff
@@ -23,6 +26,10 @@ const writeOperations: string[] = [
   'createCase',
   'deleteCase',
   'updateCase',
+  'createComment',
+  'deleteAllComments',
+  'deleteComment',
```

Comment:
Same as (comment), if we don't need to differentiate then we should consider using a single `deleteComment` operation here.